### ISSUE TITLE
Page rule options changed

### ISSUE THREAD

**Issue by tyssen** [croxton/Mustash-Cloudflare#1]
Hi Mark,

Just working through the settings and it seems CloudFlare might've changed some of the options at their end since you wrote the docs.

`Edge cache expire TTL: Respect all existing headers, Custom caching: Cache everything`

The `edge cache TTL` rule doesn't have a `respect…` option, you can only choose a time-based option and their doesn't appear to be a `custom cacheing` rule.

**Comment by croxton** [comment]
Yes there was an update. I reported the missing 'respect headers' option to Cloudflare and they acknowledged it was a bug, but they haven't added it back yet. However, the update made it the default behaviour so you don't actually need it now.

This is what you should have now, with the 'cache everything' rule _below_ the two other rules excluding admin and ACT from caching like so:

![screen shot 2016-06-15 at 10 10 10](https://cloud.githubusercontent.com/assets/279127/16074287/ab160704-32e1-11e6-8f07-a7fdf6dbe957.png)

![screen shot 2016-06-15 at 10 07 29](https://cloud.githubusercontent.com/assets/279127/16074291/b1888bc0-32e1-11e6-9665-8d9027e02b1b.png)

If you need any more rules excluding urls from caching make sure to add them above that last rule.

I also have an open ticket with Cloudflare about reducing the minimum 30 minute TTL for client-side caching, they are still looking into that.

**Comment by tyssen** [comment]
Thanks Mark. Also, having trouble finding the `mustash_cloudflare_domain_zone_id`.  I take it that's unique to each website but don't see anywhere in Cloudflare's CP where it's displayed.

**Comment by croxton** [comment]
Yes it is, and I have no idea why Cloudflare make it hard to find. After selecting a website you will find it in the page source. But probably the easiest way to find it is to open the developer console in Chrome, click on the 'Network' tab, select the 'XHR' tab and then view the response for the 'overview' request. You'll see something like:

`{"result":{"id":"12345567890","next_billing_date":null},"success":true,"errors":[],"messages":[]}`

The id value is what you want.

**Comment by tyssen** [comment]
Thanks :)
